release-team/role-handbooks/communications/templates/writing-buddy-assignment.md

```diff
@@ -0,0 +1,21 @@
+# Writing Buddy Assignement
+
+This is a message to send to the owners of the Feature Blog placeholder PRs to assign them a writing buddy.
```

Comment:
Consider saying this message should be posted as a comment in the Feature Blog PR